authorization-server/src/modules/health-check/health-check.service.ts

```diff
@@ -0,0 +1,37 @@
+import { Injectable } from '@nestjs/common';
+import {
+  HealthCheckService as TerminusHealthCheckService,
+  HealthIndicatorResult,
+} from '@nestjs/terminus';
+import { RpcHealthIndicator } from './rpc-health-indicator';
+import { IpfsHealthIndicator } from './ipfs-health-indicator';
+import { ConfigService } from '@nestjs/config';
+
+@Injectable()
+export class HealthCheckService {
+  private readonly healthChecks: (() => Promise<HealthIndicatorResult>)[];
+
+  constructor(
+    private readonly config: ConfigService,
+    private readonly terminusHealthCheckService: TerminusHealthCheckService,
+    private readonly rpc: RpcHealthIndicator,
+    private readonly ipfs: IpfsHealthIndicator,
+  ) {
+    this.healthChecks = [
+      ...(!config.get('DISABLE_HEALTHCHECK_RPC')
```

Comment:
Could you add this env var to env-validation-schema?

Reply:
done